scripts/test-setup.rs

```diff
@@ -10,12 +10,12 @@ async fn main() {
 
     let credential_provider = get_test_credential_provider();
 
-    let cache_client = CacheClient::new(
-        credential_provider,
-        configurations::laptop::latest(),
-        Duration::from_secs(5),
-    )
-    .expect("cache client cannot be created");
+    let cache_client = CacheClient::builder()
+        .default_ttl(Duration::from_secs(5))
+        .configuration(configurations::laptop::latest())
+        .credential_provider(credential_provider)
+        .build()
+        .expect("cache client cannot be created");
```

Comment:
Nit: https://doc.rust-lang.org/std/result/enum.Result.html#method.expect

> If you’re having trouble remembering how to phrase expect error messages remember to focus on the word “should” as in “env variable should be set by blah” or “the given binary should be available and executable by the current user”.
> For more detail on expect message styles and the reasoning behind our recommendation please refer to the section on [“Common Message Styles”](https://doc.rust-lang.org/std/error/index.html#common-message-styles) in the [std::error](https://doc.rust-lang.org/std/error/index.html) module docs.

Reply:
:ack:

Reply:
fixed in https://github.com/momentohq/client-sdk-rust/pull/191/commits/44fb3db09595c3ee6b41fc20aebfa954198cdaaf